AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -27,6 +27,7 @@ struct SessionHeaderView: View {
     @State var showAddNoteModal = false
     @State var showShareModal = false
     @State var showEditView = false
+    @State var detectEmailSent = false
```

Comment:
This pattern is a bit smelly, but right now I don't have a quick answer how to avoid doing this here.